Packs/DevSecOps/Integrations/GitLab/GitLab.py

```diff
@@ -184,6 +185,18 @@ def get_raw_file_request(self, project_id: str, file_path: str, ref: str):
         response = response.strip("'").strip('"')
         return response
 
+    def trigger_contrib_build(self, base_branch: str, contrib_branch: str, pr_number: int, token: str):
+        headers = self._headers
+        data = {
+            'token': token,
+            'ref': base_branch,
+            'variables[CONTRIB_BRANCH]': contrib_branch,
+            'variables[PULL_REQUEST_NUMBER]': pr_number,
+            'variables[CI_COMMIT_BRANCH]': base_branch,
+            'variables[CI_PIPELINE_SOURCE]': 'contrib'
```

Comment:
both argument names and values should be configurable by the integration arguments